### ISSUE TITLE
custom fine-tuning with DNABERT-XL

### ISSUE THREAD

**Issue by rosherbal** [jerryji1993/DNABERT#67]
Dear @jerryji1993 and @Zhihan1996,
I have enjoyed your paper and I now I am trying DNABERT in a non-classification way, but more in a BLAST-type analysis. As I want to find regions in the pre-trained DNABERT6 object that you provide with a specific sequence.
My sequence is around 2Kb, so I wanted to use DNABERT-XL.
I run over this sequence the `seq2kmer` function that you provided to calculate the 6-mer.
Then I feed it to `run_finetune.py` with only one label '0', such as
`python run_finetune.py \
    --model_type dnalong \
    --tokenizer_name=dna$KMER \
    --model_name_or_path $MODEL_PATH \
    --task_name dnaprom \
    --do_train \
    --data_dir $DATA_PATH \
    --max_seq_length 2048 \
    --per_gpu_eval_batch_size=32   \
    --per_gpu_train_batch_size=32   \
    --learning_rate 2e-4 \
    --num_train_epochs 5.0 \
    --output_dir $OUTPUT_PATH \
    --logging_steps 100 \
    --save_steps 4000 \
    --warmup_percent 0.1 \
    --hidden_dropout_prob 0.1 \
    --overwrite_output \
    --weight_decay 0.01 \
    --n_process 8`

However, it stops without finishing and no error messenger is reported. Interestingly, the code run without any problem if the `max_seq_length` is 512.
I can provide you with the log file of the run if you need it.

Also, I was trying to use another type of `task_name`, as it is not dnaprom discovery. Is there some documentation of the available options? 

Best,

Rosa

**Comment by aliakay** [comment]
up! What is this task_name?

**Comment by rosherbal** [comment]
This task_name is a parameter required to run the code, and change it to something else gives an error

**Comment by smruti241** [comment]
@rosherbal I have a query. I have unlabeled kmer data, I want to finetune with pretrained models. Can you please tell me how to run finetune script with this dataset?
how to create dev, test and train tsv files?
